examples/rollup.config.js

```diff
@@ -51,59 +59,37 @@ const staticFiles = [
 
     // modules (N.B. destination folder is 'modules' as 'node_modules' are automatically excluded by git pages)
     { src: './node_modules/monaco-editor/min/vs', dest: 'dist/modules/monaco-editor/min/vs' },
-    // N.B. fflate will not be needed once extras module is rolled up
-    { src: '../node_modules/fflate/esm/', dest: 'dist/modules/fflate/esm' }
-];
 
-const regexpExportStarFrom =  /^\s*export\s*\*\s*from\s*.+\s*;\s*$/gm;
-const regexpExportFrom     =  /^\s*export\s*{.*}\s*from\s*.+\s*;\s*$/gm;
-const regexpImport         =  /^\s*import\s*.+\s*;\s*$/gm;
-/**
- * If one of this RegExp's match, it's likely an ESM with external dependencies.
- * @example
- * isModuleWithExternalDependencies(`
- *    // Testing variants:
- *    export * from './index.mjs';
- *    export { Ray } from './core/shape/ray.js';
- *    import './polyfill/OESVertexArrayObject.js';
- *`);
- * @param {string} content - The file content to test.
- * @returns {boolean} Whether content is a module.
- */
-function isModuleWithExternalDependencies(content) {
-    const a = regexpExportStarFrom.test(content);
-    const b = regexpExportFrom.test(content);
-    const c = regexpImport.test(content);
-    // console.log('isModuleWithExternalDependencies', { a, b, c });
-    return a || b || c;
-}
+    // TODO: fflate will not be needed once extras module is rolled up
+    { src: '../node_modules/fflate/esm/', dest: 'dist/modules/fflate/esm' },
 
-const { NODE_ENV = '' } = process.env;
-let { ENGINE_PATH = '' } = process.env;
+    // engine path
+    ...getEnginePathFiles()
+];
 
-// If we don't set ENGINE_PATH and NODE_ENV is 'development', we use ../src/index.js, which
-// requires no additional build shells.
-if (!ENGINE_PATH && NODE_ENV === 'development') {
-    ENGINE_PATH = '../src/index.js';
-}
+function getEnginePathFiles() {
+    if (!ENGINE_PATH) {
+        return [];
+    }
 
-if (ENGINE_PATH) {
     const src = path.resolve(ENGINE_PATH);
     const content = fs.readFileSync(src, 'utf8');
-    const copyDir = isModuleWithExternalDependencies(content);
-    if (copyDir) {
-        // Copy entire folder for MJS versions with external dependencies
+    const isUnpacked = isModuleWithExternalDependencies(content);
+    if (isUnpacked) {
         const srcDir = path.dirname(src);
         const dest = 'dist/iframe/ENGINE_PATH';
-        staticFiles.push({ src: srcDir, dest });
-    } else {
-        // This can be both UMD/ESM as a single file
-        const entryPoint = ENGINE_PATH.split("/").pop();
-        const dest = 'dist/iframe/ENGINE_PATH/' + entryPoint;
-        staticFiles.push({ src, dest });
+        return [{ src: srcDir, dest }];
     }
+
+    // packed module builds
+    ENGINE_PATH.split("/").pop();
```

Comment:
This looks like a statement that just does nothing so far? If you still work on it, please mark it as draft until ready for review :detective:

Reply:
That line should have been removed 😅